calico_rkt/calico_rkt.py

```diff
@@ -1,91 +1,249 @@
-import sys
+#!/usr/bin/env python
```

Comment:
This should have the Metaswitch copyright at the top.  Also I don't think this line here is required.